pkg/webhook/webhook.go

```diff
@@ -556,6 +556,17 @@ func createResourcePatch(patch []jsonPatchOperation, Containers []corev1.Contain
 		patch = patchEmptyResources(patch, 0, "limits")
 	}
 
+	for resourceName, _ := range resourceRequests {
+		for _, container := range Containers {
+			if _, exists := container.Resources.Limits[corev1.ResourceName(resourceName)]; exists {
```

Comment:
should we also check resource quantities too before deleting entry from resourceRequests map ? when injected resource requests/limits < networks (i.e. pod spec has same network multiple times), in this case should we just adjust the map.entry value ?

Reply:
I think we should either inject or not inject as it's impossible for NRI to predict which container user wants to inject resource to or how many resource per container should be injected if there are multiple containers in pod spec. wdyt?

Reply:
ok, I get it, let's go with disabling all the resource injections when any one of the container's resource request/limit has the same resource requested.